ProjectLighthouse.Servers.Website/Pages/LoginForm.cshtml

```diff
@@ -64,5 +65,12 @@
                 Register
             </div>
         </a>
-    }
+    }       
+    </div>
+    <br/>
+        <a href="/passwordresetrequest">     
```

Comment:
```suggestion
        <a href="/passwordResetRequest">
```

I'd also wrap this button in a `@if` to see if email is enabled. A common UX pattern (for Lighthouse at least) is to hide buttons for things that are unusable due to configurations. For example if external auth is disabled the `Authentication` button will not show on the navbar.